README.md

```diff
@@ -224,6 +224,82 @@ permanent: true
 The permanent and runtime settings are independent, so you can set only the runtime, or only the permanent.  You cannot
 set both permanent and runtime to `false`.
 
+Examples of Options
+-------------------
```

Comment:
Since the default is `permanent: yes` and `runtime: yes` I think you can omit them from the examples, and say something like "By default, any changes will be applied immediately, and to the permanent settings.  If you want the changes to apply immediately but not permanently, use `permanent: no`.  Conversely, use `runtime: no`".